backend/src/controllers/post.controller.js

```
@@ -0,0 +1,165 @@
+		path: "comments",
+		populate: {
+			path: "user",
+			select: "username firstName lastName profilePicture",
+		},
+	});
+
+	res.status(200).json({ posts });
+});
+
+
+export const getPosts = asyncHandler(async (req, res) => {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'getPosts'. Consider to delete it or rename it.

'getPosts' is defined here:


(lint/suspicious/noRedeclare)